### DISCUSSION TITLE
blog/react_compiler_3/

### DISCUSSION THREAD

**Discussion by giscus** [yongsk0066/yongsk0066.github.io#2] [Comments]
# blog/react_compiler_3/

React Compiler의 Lowering 과정을 살펴보자. AST를 HIR로 변환하는 과정이다.

https://yongsk0066.github.io/blog/react_compiler_3/

**Comment by yongsk0066** [comment]
안녕하세요 유익한 글이네요. 댓글 테스트좀 해볼게요.

**Reply by kimjeongwonn** [reply]
감사합니다

**Comment by kimjeongwonn** [comment]
유용한 포스팅 감사합니다 ^^ 계속 연재 부탁드려요

**Reply by yongsk0066** [reply]
감사합니다. 힘이되는 댓글이네요

**Reply by kimjeongwonn** [reply]
다음화는 언제 나오나요

**Comment by HidenLee** [comment]
FECONF2025 세션 듣고 찾아보다 이 글까지 도달했습니다.
좋은 포스팅 감사합니다!

**Reply by yongsk0066** [reply]
블로그 까지 찾아와 주셨군요! 감사합니다 🙇🏽
